google/cloud/pubsublite/internal/resumable_async_streaming_read_write_rpc.h

```diff
@@ -253,7 +236,7 @@ class ResumableAsyncStreamingReadWriteRpcImpl
         });
   }
 
-  future<bool> Write(RequestType const& r, grpc::WriteOptions o) override {
+  future<bool> Write(RequestType const& r) override {
```

Comment:
This is uncontroversial, but needs to be in a separate `cleanup(pubsublite): `-type PR.

Reply:
https://github.com/googleapis/google-cloud-cpp/pull/8506